cli/deployment/flags.go

```diff
@@ -0,0 +1,447 @@
+package deployment
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/spf13/pflag"
+
+	"github.com/coder/coder/cli/cliflag"
+	"github.com/coder/coder/codersdk"
+)
+
+const (
+	secretValue = "********"
+)
+
+func NewFlags() codersdk.DeploymentFlags {
```

Comment:
What do you think about this being a `var` instead? Just like `deployment.Flags`. Nothing dynamic happens in here anyways, and it'd reduce nesting a bit.

Reply:
I don't like using vars here because you don't know if they are changed at runtime, this is safer to me.